packages/web3-shared/evm/utils/errors.ts

```diff
@@ -0,0 +1,289 @@
+const pub = getPubkeyFromPrivateKey(prv);
+const addr = getAddressFromPublicKey(pub);
+const addrPrv = getAddressFromPrivateKey(prv);
+console.log(isPrivateKey(prv));
+console.log(isPublicKey(pub));
+console.log(isAddress(addr));
+console.log(isAddress(addrPrv));
+```
+Encrypt/decrypt apis
+```javascript
+* const { Wallet } = require('@harmony-js/account');
+* const myPhrase = new Wallet().newMnemonic();
+* console.log(myPhrase);
+* const pwd = '1234';
+* encryptPhrase(myPhrase, pwd).then((value) => {
+*   console.log(value);
+*   decryptPhrase(JSON.parse(value), pwd).then(value => {
+*     console.log(value);
+*   });
+* });
+```
+ *
+ * @packageDocumentation
+ * @module harmony-crypto
+ */
+
+// This file is ported from ether.js/src.ts/errors.ts
+
+// Unknown Error
+/** @hidden */
+export const UNKNOWN_ERROR = 'UNKNOWN_ERROR'
+
+// Not implemented
+/** @hidden */
+export const NOT_IMPLEMENTED = 'NOT_IMPLEMENTED'
+
+// Missing new operator to an object
+//  - name: The name of the class
+/** @hidden */
+export const MISSING_NEW = 'MISSING_NEW'
+
+// Call exception
+//  - transaction: the transaction
+//  - address?: the contract address
+//  - args?: The arguments passed into the function
+//  - method?: The Solidity method signature
+//  - errorSignature?: The EIP848 error signature
+//  - errorArgs?: The EIP848 error parameters
+//  - reason: The reason (only for EIP848 "Error(string)")
+/** @hidden */
+export const CALL_EXCEPTION = 'CALL_EXCEPTION'
+
+// Invalid argument (e.g. value is incompatible with type) to a function:
+//   - argument: The argument name that was invalid
+//   - value: The value of the argument
+/** @hidden */
+export const INVALID_ARGUMENT = 'INVALID_ARGUMENT'
+
+// Missing argument to a function:
+//   - count: The number of arguments received
+//   - expectedCount: The number of arguments expected
+/** @hidden */
+export const MISSING_ARGUMENT = 'MISSING_ARGUMENT'
+
+// Too many arguments
+//   - count: The number of arguments received
+//   - expectedCount: The number of arguments expected
+/** @hidden */
+export const UNEXPECTED_ARGUMENT = 'UNEXPECTED_ARGUMENT'
+
+// Numeric Fault
+//   - operation: the operation being executed
+//   - fault: the reason this faulted
+/** @hidden */
+export const NUMERIC_FAULT = 'NUMERIC_FAULT'
+
+// Insufficient funds (< value + gasLimit * gasPrice)
+//   - transaction: the transaction attempted
+/** @hidden */
+export const INSUFFICIENT_FUNDS = 'INSUFFICIENT_FUNDS'
+
+// Nonce has already been used
+//   - transaction: the transaction attempted
+/** @hidden */
+export const NONCE_EXPIRED = 'NONCE_EXPIRED'
+
+// The replacement fee for the transaction is too low
+//   - transaction: the transaction attempted
+/** @hidden */
+export const REPLACEMENT_UNDERPRICED = 'REPLACEMENT_UNDERPRICED'
+
+// Unsupported operation
+//   - operation
+/** @hidden */
+export const UNSUPPORTED_OPERATION = 'UNSUPPORTED_OPERATION'
+
+// tslint:disable-next-line: variable-name
+/** @hidden */
```

Comment:
what is this jsdoc comment? if you want to hide it from `.d.ts` emitting, you should use `@internal`

Reply:
@Jack-Works this file will be deleted, I will use the package